src/ring_theory/integral_closure.lean

```diff
@@ -332,6 +299,12 @@ lemma algebra.is_integral_trans (A_int : ∀ x : A, is_integral R x)(B_int : ∀
 
 end algebra
 
+-- Why is this so slow?
```

Comment:
I worry about this comment, since the change in this PR is supposed to speed up unification. What's the issue here? How slow are we talking? (I haven't checked it out yet.)

Reply:
@kckennylau, this was one of yours, would you mind taking another look?

Reply:
4.69s elaboration

Reply:
I can't reproduce. It's no slower than anything else in the file for me, much faster than `is_integral_mem_of_fg`.

Reply:
`elaboration of integral_closure_idem took 379ms`.

Sorry, Kenny. :-(